xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/export_modal.vm

```diff
@@ -78,23 +90,23 @@
               'showRoot': true,
               'showAttachments': false,
               'showTranslations': false,
-              'readOnly': false
+              'readOnly': false,
+              'filterHiddenDocuments': false
             })
           </div>
         #end
-        <a href="$doc.getURL('export', "format=html&amp;$!escapetool.html($request.getQueryString())")" class="btn btn-primary" rel="nofollow">$services.localization.render('core.menu.export.html')</a>
+        <a href="$doc.getURL('export', "format=html")" class="btn btn-primary"
```

Comment:
You dropped the query string from the current URL. Where did it go? You don't need it at all?

Reply:
No it's now completely computed through the JS. But we might keep it, as the URL is rewritten in the JS, so I can push it back.